services/src/util/config.rs

```diff
@@ -214,7 +214,6 @@ pub struct Logging {
     pub log_spec: String,
```

Comment:
you removed the option to enable/disable buffering? What if I need to disable buffering to flush logs that are otherwise eaten up?

Reply:
Can only happen if program aborts before flush interval or full buffer.